File: libs/perllib/LoxBerry/testing/currtime.pl
#!/usr/bin/perl
use LoxBerry::System;


print "hr           : " . currtime() . "\n";
print "hrtime       : " . currtime('hrtime') . "\n";
print "hrtimehires  : " . currtime('hrtimehires') . "\n";
print "file         : " . currtime('file') . "\n";
print "filehires    : " . currtime('filehires') . "\n";
print "iso          : " . currtime('iso') . "\n";

File: libs/pythonlib/testing/currtime.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
currtime.py - Test all currtime() formats.
Analogous to perllib/LoxBerry/testing/currtime.pl.
"""

import os
import sys

sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))

from loxberry import system as lb

for fmt in ("", "hr", "hrtime", "hrtimehires", "file", "filehires", "iso"):
    print("  currtime(%-12r) = %s" % (fmt, lb.currtime(fmt)))

print("\nOK")
